core/execution.go

```diff
@@ -94,15 +101,62 @@ func exec(env vm.Environment, caller vm.ContractRef, address, codeAddr *common.A
 		}
 	}
 	env.Transfer(from, to, value)
+	snapshotPostTransfer := env.MakeSnapshot()
```

Comment:
This (and the code below) should only happen for after the homestead block. Did I miss a check somewhere?